monitor/linuxmonitor/processor.go

```diff
@@ -197,6 +142,14 @@ func (s *LinuxProcessor) Destroy(eventInfo *rpcmonitor.EventInfo) error {
 		)
 	}
 
+	if eventInfo.HostService {
+		if err := ioutil.WriteFile("/sys/fs/cgroup/net_cls,net_prio/net_cls.classid", []byte("0"), 0644); err != nil {
+			return errors.New("Failed to  write to net_cls.classid file for new cgroup")
```

Comment:
you should tell what was the original error also

Reply:
ok